templates/conf/fluent.conf.erb

```diff
@@ -78,6 +78,20 @@
   client_key "#{ENV['FLUENT_OPENSEARCH_CLIENT_KEY']}"
   client_key_pass "#{ENV['FLUENT_OPENSEARCH_CLIENT_KEY_PASS']}"
   index_name "#{ENV['FLUENT_OPENSEARCH_INDEX_NAME'] || 'fluentd'}"
+  logstash_dateformat "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_DATEFORMAT'] || '%Y.%m.%d'}"
+  logstash_format "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_FORMAT'] || 'false'}"
+  logstash_prefix "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_PREFIX'] || 'logstash'}"
+  logstash_prefix_separator "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_PREFIX_SEPARATOR'] || '-'}"
+  <buffer>
+    flush_thread_count "#{ENV['FLUENT_OPENSEARCH_BUFFER_FLUSH_THREAD_COUNT'] || '1'}"
+    flush_mode interval
```

Comment:
Could you make configurable with adding environment variable for this parameter?
Using interval is suitable for ingesting logs into OpenSearch in most cases but some of people want to change this retrying behavior.

Reply:
Thanks for the feedback. I'm on it.

Reply:
Done